js/alert_banner.js

```diff
@@ -11,34 +11,34 @@
 
   function setAlertBannerHideCookie(cookie_tokens, token) {
     cookie_tokens.push(token);
-    var new_cookie = cookie_tokens.join('+');
+    const new_cookie = cookie_tokens.join('+');
     // Set expiry 30 days.
-    var expiry = Date.now() + (30 * 24 * 60 * 60 * 1000);
+    const expiry = Date.now() + (30 * 24 * 60 * 60 * 1000);
     document.cookie = 'hide-alert-banner-token=' + new_cookie + '; expires=' + new Date(expiry).toString() + '; SameSite=Lax;'
   }
 
   $(document).ready(function() {
 
-    var all_cookies = document.cookie.split('; ');
-    for (var i = 0; i < all_cookies.length; i++) {
-      var indv_cookie = all_cookies[i].split('=');
+    const all_cookies = document.cookie.split('; ');
+    for (const i = 0; i < all_cookies.length; i++) {
```

Comment:
Hi Mark, this is raising an exception in the Javascript console as `i` here needs to be a variable rather than const :(

Reply:
That's updated now.